### DISCUSSION TITLE
posts/024-hifi-test/

### DISCUSSION THREAD

**Discussion by giscus** [Yamabuki-bakery/tasty-discussion#5] [Announcements]
# posts/024-hifi-test/

使用妳最喜歡的網易雲歌曲來鑑定自己是不是木耳！！

**Comment by PermeationLoop** [comment]
用充话费送的耳機測的，低難度模式

<img width="461" height="463" alt="image" src="https://github.com/user-attachments/assets/57d9dc81-fd46-4b0d-8258-abc95ee1874e">

<br />

訣竅是選一些人聲動態範圍比較大，樂器等變化也比較大的歌曲來測試，然後注意聽各種聲音混合的片段……對不起我編不下去了，選近年的歌曲也許區別會更明顯？

**Comment by PermeationLoop** [comment]
根據 GPT 所述，

常用顯著性水平：在大多數科學研究中，常用的顯著性水平是 0.05，這意味著你有 5% 的機會錯誤地拒絕真實的零假設（即第一類錯誤）1。
綜合考慮這些因素，對於你的應用程序，建議使用 0.05 的顯著性水平，這是大多數情況下的標準選擇，能夠提供合理的準確性和可靠性。

所以俺把顯著性水平調回了 0.05（原來放水了，是 0.06）；這種情況下，進行十次試聽必須要答對 9 次纔能通過測試（當然聽得越多測得越準

**Reply by PermeationLoop** [reply]
因此如果妳之前測得的結果低於 5%，那就是通過了；之前畫的圖表有點問題，現在修好了😁👍🏻

**Reply by PermeationLoop** [reply]
所以俺 13 次裏面對 10 次，仍然通過

**Comment by erikopak-cmyk** [comment]
我很喜歡淘寶賣12.9元人民幣的原道耳機

**Reply by PermeationLoop** [reply]
那妳通過這個測試了嗎 🥰？
